lib/charms/opensearch/v0/opensearch_health.py

```diff
@@ -133,6 +134,8 @@ def get(  # noqa: C901
     @retry(stop=stop_after_attempt(90), wait=wait_fixed(10), reraise=True)
     def wait_for_shards_relocation(self) -> None:
         """Blocking function until the shards relocation completes in the cluster."""
+        time.sleep(5)
```

Comment:
I have added this change to the `wait_for_shards_relocation` method because I saw error 503 still occurring randomly in integration tests. My assumption is, that checking the shards relocation immediately after adding the allocation exclusion might be to fast for Opensearch, and relocation may not have been started yet. Tbh. five seconds is just a guess, but I think in the case of shutdown, this extra wait is acceptable.

Reply:
I'm not sure it will help, given we already have the `wait_fixed(10)` in the tenacity decorator.

Reply:
I'm afraid the `wait` only applies on retries. If the health is green on the first attempt (because it hasn't started relocating yet), it will go through without waiting.